### PR TITLE
EG-4127: restructuring the config information to make it easier to un…

### PR DESCRIPTION
**Pull request by AMcFarlandCampbellR3**
`content/en/docs/corda-enterprise/4.6/operations/deployment/corda-firewall-configuration-file.md` and `content/en/docs/corda-enterprise/4.6/operations/deployment/firewall-deployment.md` were identical. The original content was very long, so I split the content across the two files - the former had a restructured configuration section, and the latter had everything else. No content has been removed.

One reason for splitting the files like this is the main configuration keys are now `h2` sections, so they appear in the RHS menu.

There was also a minor change made to `content/en/docs/corda-enterprise/4.6/node/corda-firewall-configuration-file.md`. It was identical to the other two files. It will be updated (split into two sections) as part of the work on EG-4181.

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.6/operations/deployment/corda-firewall-configuration-file.md [hunk 1]
```diff
@@ -9,1356 +9,311 @@ tags:
...
 
-```bash
-corda-firewall [-hvV] [--install-shell-extensions]
-               [--logging-level=<loggingLevel>] [-b=<baseDirectory>]
-               [-f=<_configFile>]
-```
+## firewallMode
+Determines operating mode of the firewall. See [Firewall operating modes](firewall-deployment.md#firewall-operating-modes).
 
-Where:
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.6/operations/deployment/corda-firewall-configuration-file.md:31]: @AMcFarlandCampbellR3 Probably a question that I will find the answer to later in the PR but - is this content going way? Same for the code example above?

Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.6/operations/deployment/corda-firewall-configuration-file.md:31]: `corda-firewall-configuration-file.md` (this file) and `firewall-deployment.md` were identical. The original content was very long, so I split the content across the two files - the former had a restructured configuration section, and the latter had everything else. No content has gone. This was obvious when working on the document, but it isn't so obvious from the PR, so I will update the description.

**Review by dimosr — APPROVED** [review]
I did a quick scan, but didn't read every field description in detail since I assumed it's mostly moving them around and not substantial changes content-wise. I also did a local build to see how it looks like and it's much better!

One thing that might make it even better is making use of nested lists, since there are some inconsistencies now that might confuse the reader just a bit. For example, lists and sub-headings are used interchangeably in some cases for nested fields.

**Comment by AMcFarlandCampbellR3** [comment]
> One thing that might make it even better is making use of nested lists, since there are some inconsistencies now that might confuse the reader just a bit. For example, lists and sub-headings are used interchangeably in some cases for nested fields.

The ideal solution would be to have each key as a top-level heading in the document, with children listed in a bulleted list underneath. In this case, sometimes the child keys had different descriptions depending on what parent the had. That is why we ended up with the somewhat hybrid solution. Fixing that to make everything consistent could have taken a very long time.

In the medium term, I think we need to move to documenting all HOCON config files in the same standardised way, and I will be developing that standard based, in part, on the work I did on this ticket.

**Review by taylorharvey-r3 — APPROVED** [review]
LGTM

**Pull request merged**